app/routes/docs_.schema.tsx

```diff
@@ -73,6 +76,9 @@ export default function Schema() {
         <BreadcrumbBar className="tablet:grid-col-fill">
           <>{...breadCrumbs}</>
         </BreadcrumbBar>
+        {windowSize.width < 480 && !isSchema && (
+          <h2 className="margin-y-0">{path.split('/').slice(-1)[0]}</h2>
+        )}
```

Comment:
Can we please show the full cookie crumb path, even on mobile?

Reply:
We do on larger sizes, but the USWDS component drops all but the previous nav point on smaller screens